pkg/features/kube_features.go

```diff
@@ -201,12 +201,6 @@ const (
 	// Enable Block volume support in containers.
 	BlockVolume utilfeature.Feature = "BlockVolume"
 
-	// owner: @pospispa
-	// beta: v1.10
-	//
-	// Postpone deletion of a PV or a PVC when they are being used
-	StorageObjectInUseProtection utilfeature.Feature = "StorageObjectInUseProtection"
```

Comment:
Removal of a gate breaks startups that explicitly enabled it. I'd expect this to remain, with GA status, either indefinitely, or for an announced deprecation period